project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -54,9 +54,25 @@
 
   /**
    * @return the coordinates of the physical place.
-  */
+   */
   public abstract LatLng location();
 
+  /**
+   * @return the URL of the place’s google page.
+   */
+  public abstract String googleUrl();
+
+  /**
+    * @return the place's placeId as defiened on Google Places.
+    */
+  public abstract String placeId();
+
+  /**
+    * @return the place's business status, represented by one of the following:
```

Comment:
I have a semi technical question - can an enum be in a class that's not named after it? 
I that's possible, maybe we should put the enum in the util class for Place - Places.java?

Reply:
Yes it is possible, it can be defined on the Places class for example and then approached by Places.BusinessStatus 
I saw online that the best practice for public enums is to define them on a seperate file since it is just like a class, and that makes sense to me.

Reply:
I agree, this way it can be shared among different modules in the future, without them needing to know some util file.